esmvaltool/cmorizers/data/formatters/datasets/ceres_ebaf.py

```diff
@@ -12,7 +12,6 @@
 Download and processing instructions
```

Comment:
This script is currently not working because of the call to the cmorizer function.

Reply:
Fixed